src/ServiceProviders/PaymentGateways.php

```diff
@@ -19,21 +21,25 @@ class PaymentGateways implements ServiceProvider {
 	 * @var string[]
 	 */
 	public $gateways = [
+		PayPalStandard::class,
 		PayPalCheckout::class,
 	];
 
 	/**
 	 * @inheritDoc
 	 */
 	public function register() {
-		// Not used, but needed for interface
 	}
 
 	/**
 	 * @inheritDoc
 	 */
 	public function boot() {
 		add_filter( 'give_payment_gateways', [ $this, 'registerGateways' ] );
+
+		// Register paypal setting section.
+		$paypalSettingPage = new PaypalSettingPage();
+		$paypalSettingPage->register()->boot();
```

Comment:
Since settings only happen on the admin-side, let's place this within a function that gets called on the `admin_init` WP action. That way these don't get instantiated on the front-end.

Also, since this class is using an interface, let's put it in a `private $gatewaySettingsPages` array that gets looped through, instantiated, and initiated.

Reply:
fixed: https://github.com/impress-org/givewp/compare/ab0e47973f26...64173ddb3cee https://github.com/impress-org/givewp/compare/64173ddb3cee...d9eaaa7c2f93 https://github.com/impress-org/givewp/compare/d9eaaa7c2f93...8f8e9bd44851